libs/output-mapping/src/Writer/Helper/SliceCommandBuilder.php

```diff
@@ -19,7 +19,7 @@ public static function createProcess(
         ?string $inputSizeThreshold = null,
     ): Process {
         $command = [
-            './bin/slicer',
```

Comment:
https://getcomposer.org/doc/articles/vendor-binaries.md#finding-the-composer-bin-dir-from-a-binary